admin_guide/configuring_authentication.adoc

```diff
@@ -464,7 +464,7 @@ the Apache VirtualHost:
     provider:
       apiVersion: v1
       kind: RequestHeaderIdentityProvider
-      clientCA: /etc/openshift/master/ca.crt
+      clientCA: /etc/openshift/master/proxy/ca.crt
```

Comment:
Cool, yeah that was probably just an oversight.  I didn't realize `/etc/openshift/master/proxy/ca.crt` was a side effect of `create-api-client-config`.  It seems a little weird to me to create `proxyca.crt` in the `create-signer-cert` command and then not use it.  I'm sure someone will be confused either way though since it's inherently confusing to have two files with the same contents.  I'll let you choose whichever one you like the most.  The rest of the PR looks good to me.